src/pages/validators/node.mdx

```diff
@@ -40,44 +83,54 @@ This is needed to store ZetaChain binary and configuration files.
 
 ```bash
 sudo su zetachain
-mkdir -p /home/zetachain/.zetacored/bin
 mkdir /home/zetachain/.zetacored/config
 ```
 
-## Create a Systemd Unit File
+### Configure a Systemd Unit File
 
-We recommend using Systemd to start and stop ZetaChain binary files and view the
-logs. You can create a Systemd unit file at the following location
-`/etc/systemd/system/zetacored.service`:
+Create the file `/etc/systemd/system/zetacored.service` with the following contents:
 
-```bash
+```
 [Unit]
-Description=Zetacored Service
+Description=zetacored (running under cosmovisor)
 After=multi-user.target
 StartLimitIntervalSec=0
 [Install]
 WantedBy=multi-user.target
 [Service]
+WorkingDirectory=/home/zetachain/.zetacored/cosmovisor
+Environment="DAEMON_HOME=/home/zetachain/.zetacored"
+Environment="DAEMON_NAME=zetacored"
+Environment="DAEMON_ALLOW_DOWNLOAD_BINARIES=true"
+Environment="DAEMON_RESTART_AFTER_UPGRADE=true"
+Environment="DAEMON_DATA_BACKUP_DIR=/home/zetachain/.zetacored"
+Environment="CLIENT_DAEMON_NAME=zetaclientd"
+Environment="CLIENT_SKIP_UPGRADE=false"
+Environment="CLIENT_START_PROCESS=false"
+Environment="UNSAFE_SKIP_BACKUP=true"
 Type=simple
 Restart=always
-RestartSec=90
+RestartSec=600
 User=zetachain
 LimitNOFILE=262144
-ExecStart=zetacored start --home /home/zetachain/.zetacored/ --log_format json  --log_level info --moniker <YOUR_NODE_NAME_HERE>
+ExecStart=cosmovisor run start --home /home/zetachain/.zetacored/ --log_format json  --moniker <YOUR_NODE_NAME_HERE>
 ```
 
-Then start the service with `systemctl start zetacored`.
+<Alert variant="danger">
+    In this example, we have set `DAEMON_ALLOW_DOWNLOAD_BINARIES=true`. This instructs `cosmovisor` to automatically 
+    download the new binaries specified in the upgrade proposal. This would fail if the github api is down.
 
-You can view the logs with `journalctl -o cat -f -u zetacored`.
+    We suggest keeping this feature enabled, but preloading and validating any new binaries before the upgrade to
+    minimize risk. If you have already preloaded the binaries, none will be downloaded.
 
-## CLI
+    Read more about [cosmovisor Auto-Download here](https://docs.cosmos.network/v0.47/build/tooling/cosmovisor#auto-download).
+</Alert>
 
-ZetaChain Core is built with the Cosmos SDK. There is an `--help` flag that can
-be used with any subcommand to learn its use and syntax.
+Run `systemctl daemon-reload` to load the service.
 
-```bash
-zetacored --help
-```
+Run `systemctl enable --now zetacored` to start the service and to automatically start it at boot.
+
+## 
```

Comment:
are ## intended?

Reply:
No, I meant to have another section here about how to install upgrades.